modules/alloy/src/main/ts/ephox/alloy/api/ui/Toolbar.ts

```diff
@@ -26,9 +26,9 @@ const factory: CompositeSketchFactory<ToolbarDetail, ToolbarSpec> = (detail, com
 
   const getGroupContainer = (component: AlloyComponent) => detail.shell ? Optional.some(component) : AlloyParts.getPart(component, detail, 'groups');
 
-  const refresh = (toolbar: AlloyComponent) => {
-    const toolbarApi: ToolbarApis = toolbar.getApis();
-    if (toolbarApi.refresh) {
+  const refresh = (toolbar: AlloyComponent | undefined) => {
```

Comment:
From were is this called with no arguments? Is that a mistake since then this function would be an noop?

Reply:
I ended up replacing it with a `Fun.noop`, since that `refresh` was referencing itself, let me know if you think it could make problem